### PR TITLE
content for estimations.md

### PR DESCRIPTION
**Pull request by jandriescher**
I wrote something about storypoint and estimations in the corresponding file. It includes a summary of what estimations do for us and an overview of the different possible storypoints.

### CODE REVIEW

**Review by Chemaclass — APPROVED** [review]
Thanks! 👍 🎉

**Review by JesusValeraDev — APPROVED** [review]
Niceee :+1: :1st_place_medal:

File: team-work/estimations.md [hunk 1]
```diff
@@ -9,3 +9,53 @@ Estimations can be done in different ways. You can estimate time or complexity.
...
+
+Still easily achievable without any kind of investigation but requires a bit more professional effort/ knowledge to finish.
+
+**example:** Adding a new field in a existing API resource.
+
+### 3
+
+A task that requires a certain level of knowledge or investigation to solve but will be pretty straight forward for a person that is knowladgable within the specific system/s.
+
+**example:** Adding a new configuration to a system by reusing und changing existing code.
```
Comment (JesusValeraDev) [inline team-work/estimations.md:37]: Und or and? :P
```
**example:** Adding a new configuration to a system by reusing and changing existing code.
```

File: team-work/estimations.md [hunk 2]
```diff
@@ -9,3 +9,53 @@ Estimations can be done in different ways. You can estimate time or complexity.
...
+
+### 5
+
+This task/ problem may involve multiple systems and/or substantial investigation in order to complete it. Will most likely take more then a day and usually require a decent amount of implementational work.
```
Comment (JesusValeraDev) [inline team-work/estimations.md:41]: `task/ problem` or `task /problem` or `task/problem` or `task / problem`?

**Review by JesusValeraDev — COMMENTED** [review]
Niceee :+1: :1st_place_medal:

**Pull request merged**